browser/brave_stats/first_run_util.cc

```diff
@@ -37,4 +46,18 @@ base::Time GetFirstRunTime(PrefService* local_state) {
 #endif  // #BUILDFLAG(IS_ANDROID)
 }
 
+bool IsFirstRun(PrefService* local_state) {
+#if BUILDFLAG(IS_ANDROID)
+  if (g_is_first_run) {
+    return true;
+  }
+  GetFirstRunTime(local_state);
+  return g_is_first_run;
```

Comment:
Maybe just `return false` here for clarity? Up to you.

Reply:
`g_is_first_run` is set by `GetFirstRunTime` which may be true or false

Reply:
OK, but I think we already checked the case where it's `true` just above here. Still, I'm fine with it the way it is too.

Reply:
Or do you mean `GetFirstRunTime` may set it as well? In that case, yeah leave it.

Reply:
> Or do you mean `GetFirstRunTime` may set it as well? In that case, yeah leave it.

yep, exactly